web/app/Http/Middleware/RedirectIfAuthenticated.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Wikijump\Http\Middleware;
+
+use Wikijump\Providers\RouteServiceProvider;
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class RedirectIfAuthenticated
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @param  string|null  ...$guards
+     * @return mixed
+     */
+    public function handle(Request $request, Closure $next, ...$guards)
+    {
+        $guards = empty($guards) ? [null] : $guards;
```

Comment:
Maybe leave a comment why it needs to have `null` instead of being purely empty?